EIPS/eip-meta-hardfork-meta.md

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
```suggestion
---
eip: 7568
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.